doc/requirements.txt

```diff
@@ -1,4 +1,6 @@
-numpydoc
-ipython
-nbsphinx
-sphinx_rtd_theme
+numpydoc==1.1.0
+ipython==7.17.0
+Pygments>=2.6
+nbsphinx==0.7.1
+sphinx==3.2.1
+sphinx_rtd_theme==0.5.0
```

Comment:
Pinning `Pygments>2.6` only should suffice. The other dependency versions should be okay..

Reply:
May be keep the `sphinx==3.2.1`